tools/rosbag/package.xml

```diff
@@ -31,6 +31,7 @@
   <run_depend>genmsg</run_depend>
   <run_depend>genpy</run_depend>
   <run_depend>python-crypto</run_depend>
```

Comment:
I assume this dependency can be removed when only one API is used?

Reply:
Yes it can, I only need to add the keys to the rosdistro.